cli/args/config_file.rs

```diff
@@ -357,13 +362,30 @@ impl SerializedLintConfig {
   }
 }
 
+pub trait ContainsFilesConfig {
```

Comment:
Is there a way we could get rid of this trait? It looks like we could just pass the `FilesConfig` struct around instead of having this.